src/hooks/usePropsGetters/useAddToFavoritesButtonProps.ts

```diff
@@ -1,9 +1,11 @@
 import { useState, useCallback } from 'react';
+import { QuizResultData } from '@constructor-io/constructorio-client-javascript/lib/types';
 import { GetAddToFavoritesButtonProps, QuizEventsReturn } from '../../types';
 import { Favorited } from '../../components/ResultFavoritesButton/ResultFavoritesButton';
 
 export default function useAddToFavoritesButtonProps(
-  addToFavorites: QuizEventsReturn.AddToFavorites
+  addToFavorites: QuizEventsReturn.AddToFavorites,
+  favoriteItems?: string[]
```

Comment:
I personally do not have a preference and I don't know which one is the more grammatically correct, but we had `favorited` vs `favorite` (new). Both of them sound correct to me tbh